spec/shared_contexts/with_stubbed_application.rb

```diff
@@ -6,9 +6,10 @@
   let(:means_details) { { income_details:, outgoings_details:, capital_details: } }
   let(:partner_included?) { false }
   let(:case_type) { 'appeal_to_crown_court' }
+  let(:fixture_name) { 'application' }
 
   let(:application_data) do
-    JSON.parse(LaaCrimeSchemas.fixture(1.0).read).deep_merge(
+    JSON.parse(LaaCrimeSchemas.fixture(1.0, name: fixture_name).read).deep_merge(
       'client_details' => { 'partner' => { 'is_included_in_means_assessment' => partner_included? } },
       'means_details' => means_details.deep_stringify_keys,
       'case_details' => { 'case_type' => case_type }
```

Comment:
Hm by making the fixture name amendable, do you risk adding application data that you shouldn't be in the block below?